core/src/main/java/de/danoeh/antennapod/core/event/ShowRemainTimeUpdateEvent.java

```diff
@@ -0,0 +1,7 @@
+package de.danoeh.antennapod.core.event;
+
+public class ShowRemainTimeUpdateEvent {
+    public ShowRemainTimeUpdateEvent() {
```

Comment:
I think it would be better to use an `ItemUpdatedEvent` like for the "prefer streaming" preference. We already have a ton of events that need to be handled in all list fragments that just do the same everywhere. I think we could even remove some of the existing events in the future.